### PR TITLE
More import options

### PR DESCRIPTION
**Pull request by Hagellach37**
We can offer more options in import now:
* verification count between 3 and 10
* custom tileserver
* set zoomlevel

### CODE REVIEW

File: import.html [hunk 1]
```diff
@@ -259,14 +247,21 @@ <h1 class="hero-heading">Add Your Project</h1>
...
                 </div>
+                <div class="faq-container">
+                    <div class="faq-title">Custom Tileserver URL</div>
+                    <p class="faq-text">Make sure you have permission. Add a custom tileserver URL that uses {z}, {x} and {y} as placeholders and that already includes the api key.
+                        <br/><input type="text" id="customTileServerUrl" style="color:#212121"/>
+                    </p>
+                </div>
                 <div class="faq-container">
                     <div class="faq-title">Enter Description</div>
-                    <p class="faq-text">Enter the name of your project (15 chars max)
```
Comment (ericboucher) [inline import.html]: No more name?

Comment (Hagellach37) [reply import.html]: We still have a name. :) But I seemed as if the text for the field "description" was just copied from the text for the field "name".

File: import.html [hunk 2]
```diff
@@ -98,15 +98,14 @@
...
                 progress: 0,
-
             }
-
-
            importRef.push().set({
                 project: project,
                 kml: kml,
                 key: submissionKey,
+				zoomlevel: zoomlevel,
```
Comment (mathcass) [inline import.html]: It would probably improve readability to keep this at the same intent level as other variables here.

File: import.html [hunk 3]
```diff
@@ -259,14 +247,21 @@ <h1 class="hero-heading">Add Your Project</h1>
                         <br/><select id="tileServer">
                             <option value="bing">bing</option>
                             <option value="digital_globe">digital_globe</option>
-                            <option value="google">google</option>
+                            <!-- <option value="google">google</option> -->
+							<option value="custom">custom</option>
```
Comment (mathcass) [inline import.html]: Another slight nitpick about indentation here. It's usually best to have them line up.

File: import.html [hunk 4]
```diff
@@ -259,14 +247,21 @@ <h1 class="hero-heading">Add Your Project</h1>
                         <br/><select id="tileServer">
                             <option value="bing">bing</option>
                             <option value="digital_globe">digital_globe</option>
-                            <option value="google">google</option>
```
Comment (mathcass) [inline import.html]: What's the reason for removing Google here? Should we just remove the line outright rather than commenting it out?

**Pull request merged**